src/pymor/reductors/basic.py

```diff
@@ -39,10 +39,18 @@ class ProjectionBasedReductor(BasicObject):
     check_tol
         If `check_orthonormality` is `True`, the numerical tolerance with which the checks
         are performed.
+    assemble_error_estimate
+        If `True`, assembles error estimate
+    assemble_output_error_estimate
+        If `True`, assembles output error estimate. `assemble_error_estimate` must be `True`.
```

Comment:
This wording fells strange, as you rightly set `assemble_error_estimate` to `True` if `assemble_output_error_estimate` is. I would rather go along something like

```suggestion
        If `True`, assembles output error estimate (implies `assemble_error_estimate=True`).
```